Packs/FeedInfobloxThreatIntelligence/Integrations/FeedInfobloxThreatIntelligence/FeedInfobloxThreatIntelligence.yml

```diff
@@ -0,0 +1,400 @@
+category: Data Enrichment & Threat Intelligence
+sectionOrder:
+- Connect
+- Collect
+commonfields:
+  id: InfobloxThreatIntelligenceFeed
+  version: -1
+configuration:
+- section: Connect
+  display: ""
+  displaypassword: Service API Key
+  name: api_key
+  type: 9
+  required: true
+  hiddenusername: true
+  additionalinfo: 'Service API key for Infoblox TIDE API authentication'
+- section: Collect
+  display: Fetch indicators
+  name: feed
+  type: 8
+  required: false
+  defaultvalue: "true"
+- section: Collect
+  display: Indicator Types
+  name: indicator_types
+  type: 16
+  required: false
+  options:
+  - IP
+  - HOST
+  - URL
+  - EMAIL
+  - HASH
+  defaultvalue: IP,HOST,URL,EMAIL,HASH
+  additionalinfo: The type of indicators to be retrieved.
+- section: Collect
+  display: First Fetch Time
+  name: first_fetch
+  type: 0
+  required: false
+  defaultvalue: "1 hour"
+  additionalinfo: "The date or relative timestamp from where to start fetching indicators.\n\nSupported formats: 2 minutes, 2 hours, 2 days, 2 weeks, 2 months, 2 years, yyyy-mm-dd, yyyy-mm-ddTHH:MM:SSZ\n\nFor example: 01 Mar 2025, 01 Mar 2025 04:45:33, 2025-05-17T04:45:33Z\n\nNote: The maximum allowed relative time is 4 hours or 240 minutes."
+- section: Collect
+  display: Max Indicators Per Fetch
+  name: max_fetch
+  type: 0
+  required: false
+  defaultvalue: "1000"
+  additionalinfo: "The maximum number of indicators to fetch in each run.\n\nNote: The maximum allowed value is 50000."
+- section: Collect
+  display: DGA Threat
+  name: dga_threat
+  type: 15
+  required: false
+  options:
+  - 'Yes'
+  - 'No'
+  additionalinfo: Filter the indicators having threats originated from dynamically generated algorithms.
+- section: Collect
+  display: Threat Classes
+  name: threat_classes
+  type: 16
+  required: false
+  options:
+  - APT
+  - Bot
+  - CompromisedDomain
+  - CompromisedHost
+  - Cryptocurrency
+  - DDoS
+  - DNSTunnel
+  - ExploitKit
+  - ICS
+  - IllegalContent
+  - InternetInfrastructure
+  - IntrusionAttempt
+  - LimitedDistro
+  - Malicious
+  - MaliciousNameserver
+  - MalwareC2
+  - MalwareC2DGA
+  - MalwareDownload
+  - Parked
+  - Phishing
+  - Policy
+  - PolicyViolation
+  - Proxy
+  - Scam
+  - Sinkhole
+  - Spambot
+  - Suspicious
+  - UncategorizedThreat
+  - Undefined
+  - UnwantedContent
+  - WebAppAttack
+  - Whitelist
+  additionalinfo: Filters the indicators according to the selected threat classes.
+- section: Collect
+  display: Data Providers
+  name: data_provider_profiles
+  type: 16
+  required: false
+  options:
+  - IID
+  - AISCOMM
+  additionalinfo: Filter indicators by data provider profiles.
+- section: Collect
+  display: Indicator Reputation
+  name: feedReputation
```

Comment:
Several configuration parameter names do not follow the `snake_case` convention. Please rename them for consistency.